dataplane/sdk/vppagent/mac_addrress_generator.go

```diff
@@ -0,0 +1,65 @@
+package forwarder
+
+import (
+	"context"
+	"crypto/rand"
+	"fmt"
+
+	"github.com/golang/protobuf/ptypes/empty"
+
+	"github.com/networkservicemesh/networkservicemesh/controlplane/api/connectioncontext"
+	"github.com/networkservicemesh/networkservicemesh/controlplane/api/crossconnect"
+	"github.com/networkservicemesh/networkservicemesh/dataplane/api/dataplane"
+)
+
+func NewDestinationMacAddressGenerator() dataplane.DataplaneServer {
```

Comment:
TODO: do not use this after the problem https://github.com/ligato/vpp-agent/issues/1525 will be solved